src/listeners/MainListener.js

```diff
@@ -11,6 +13,25 @@ module.exports = class MainListener extends EventListener {
   onReady () {
     this.user.setPresence({ game: { name: `@${this.user.username} help` } })
 
+    const presences = [
```

Comment:
maybe the prefix should be in all, it's more intuitive

Reply:
can be done, will do after getting my new phone